modules/olm-filtering-fbc.adoc

```diff
@@ -13,7 +13,14 @@ endif::[]
 [id="olm-filtering-fbc_{context}"]
 = Updating or filtering a file-based catalog image
 
-You can use the `opm` CLI to update or filter (also known as prune) a catalog image that uses the file-based catalog format. By extracting and modifying the contents of an existing catalog image, you can update, add, or remove one or more Operator package entries from the catalog. You can then rebuild the image as an updated version of the catalog.
+You can use the `opm` CLI to update or filter (also known as prune) a catalog image that uses the file-based catalog format. By extracting the contents of an existing catalog image, you can modify the catalog as needed, for example:
```

Comment:
IMHO, perens here are good.  `prune` is the old terminology, but deprecated, so the callback should help the reader connect the old concept with the new operation.

Reply:
Ended up removing here since we have another hit on "prune" in a NOTE a few lines down, so it will at least come up in search.